packages/client/components/PokerSidebarEstimateSection.tsx

```diff
@@ -43,10 +43,10 @@ const Title = styled('div')({
 })
 
 const Subtitle = styled('div')({
+  color: PALETTE.SLATE_500,
   fontSize: 11,
   fontWeight: 600,
-  color: PALETTE.SLATE_500,
-  lineHeight: '8px'
+  lineHeight: '12px'
```

Comment:
This only increase the subnav item height from `44` to `48` pixels, but I think it gives it room to breath. I don’t want to start getting cramped in the UI because then we’ll do it everywhere.